### PR TITLE
Hotfix/single line logs

### PR DESCRIPTION
**Pull request by sogoiii**
## Describe Changes

I made this better by doing ...

- Changed some log text
- Removed logs from api service
- fixed issue where address was used as part of tag
- cleanRequest log now removes new lines and is no longer a json object stringify

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Code styles have been enforced
- [ ] I have checked eslint

### CODE REVIEW

File: src/commands/indexer/index.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default class Indexer extends HealthCheck {
     this.environment = environment
 
     if (this.environment === Environment.localhost || this.environment === Environment.experimental) {
-      this.log(`Skiping API authentication for ${Environment[this.environment]} environment`)
+      this.log(`Skipping API authentication for ${Environment[this.environment]} environment`)
```
Comment (alexanderattar) [inline src/commands/indexer/index.ts:82]: Nice catch 😅

File: src/commands/indexer/index.ts [hunk 2]
```diff
@@ -1026,12 +1027,16 @@ export default class Indexer extends HealthCheck {
             const response = await this.apiService.updateCrossChainTransactionStatus(input)
             this.networkMonitor.structuredLog(
               network,
-              this.apiColor(`API: Cross chain transaction ${jobHash} mutation response ${JSON.stringify(response)}`),
+              this.apiColor(
+                `API: Cross chain message transaction ${jobHash} mutation response ${JSON.stringify(response)}`,
```
Comment (alexanderattar) [inline src/commands/indexer/index.ts:1031]: I wonder if these logs with `JSON.stringify` will go on multiple lines in DD and look like errors. I tried looking in DD, but couldn't find any logs that matched this line for some reason

File: src/services/api-service.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ import {
 } from '../types/api'
 import {AbstractError} from '../types/errors'
 import {StructuredLogInfo} from '../types/interfaces'
-import {cleanRequest} from '../utils/utils'
+import {cleanRequest } from '../utils/utils'
```
Comment (alexanderattar) [inline src/services/api-service.ts]: typo?

File: src/utils/utils.ts [hunk 4]
```diff
@@ -167,7 +167,7 @@ export const utf8ToBytes32 = (str: string): string => {
 }
 
 // turns multi-line query into single line and removes extra spaces
-export const cleanRequest = (query: any) => JSON.stringify(query).replace(/\s+/g, ' ')
+export const cleanRequest = (query: any) => query.replace(/\n+ /g, '').replace(/\s+ /g, ' ')
```
Comment (alexanderattar) [inline src/utils/utils.ts:170]: oh Nice

**Review by alexanderattar — APPROVED** [review]
Looks good

File: src/commands/indexer/index.ts (hunk 2 above)
Comment (sogoiii) [reply src/commands/indexer/index.ts:1031]: they are single lined. just super long.

File: src/services/api-service.ts (hunk 3 above)
Comment (sogoiii) [reply src/services/api-service.ts]: Fixed. This is why the prettier check failed.

**Pull request merged**